README.md

```diff
@@ -1,21 +1,18 @@
+## Repository navigation
 
-Details of the purpose and any published outputs from this project can be found at the link above.
+-   If you are interested in how we defined our code lists, look in the [`codelists`](./codelists) folder.
 
-The contents of this repository MUST NOT be considered an accurate or valid representation of the study or its purpose. 
-This repository may reflect an incomplete or incorrect analysis with no further ongoing work.
-The content has ONLY been made public to support the OpenSAFELY [open science and transparency principles](https://www.opensafely.org/about/#contributing-to-best-practice-around-open-science) and to support the sharing of re-usable code for other subsequent users.
-No clinical, policy or safety conclusions must be drawn from the contents of this repository.
+-   Analyses scripts are in the [`analysis`](./analysis) directory:
 
-# About the OpenSAFELY framework
+    -   If you are interested in how we defined our variables, we use the variable script [variable_helper_fuctions](analysis/variable_helper_functions.py) to define functions that generate variables. We then apply these functions in [variables_cohorts](analysis/variables_cohorts.py) to create a dictionary of variables for cohort definitions, and in [variables_dates](analysis/variables_dates.py) to create a dictionary of variables for calculating study start dates and end dates.
+    -   If you are interested in how we defined study dates (e.g., index and end dates), these vary by cohort and are described in the protocol. We use the script [dataset_definition_dates](analysis/dataset_definition_dates.py) to generate a dataset with all required dates for each cohort. This script imported all variables generated from [variables_dates](analysis/variables_dates.py).
+    -   If you are interested in how we defined cohorts, we use dataset definition script [dataset_definition_cohorts](analysis/dataset_definition_cohorts.py) to define the function that generate cohort. This script imported all variables generated from [variables_cohorts](analysis/variables_cohorts.py). We extracted the respective index dates and end dates for each cohort and applied this function to generate three specific cohorts—pre-vaccination, vaccinated, and unvaccinated—found in [dataset_definition_prevax](analysis/dataset_definition_prevax.py), [dataset_definition_vax](analysis/dataset_definition_vax.py), and [dataset_definition_unvax](analysis/dataset_definition_unvax.py). Extracted data is then combined to create our final cohorts, in the [preprocess data script](analysis/preprocess_data.R).
+    -   This directory also contains all the R scripts that process, describe, and analyse the extracted data.
 
-The OpenSAFELY framework is a Trusted Research Environment (TRE) for electronic
-health records research in the NHS, with a focus on public accountability and
-research quality.
+-   The [`lib/`](./lib) directory contains a list of active analyses.
```

Comment:
Specify which file as we will add more stuff to the lib directory as we go.

Reply:
Done: 
The [active_analyses](lib/active_analyses.rds) contains a list of active analyses.